styles/widgets/common/drawer.less

```diff
@@ -73,6 +73,14 @@
     }
 }
 
+.dx-rtl .dx-drawer-slide,
```

Comment:
This styles are specific for the Drawer strategy that uses Overlay. So let's specify it:

```css
.dx-rtl .dx-drawer-overlap.dx-drawer-slide,
.dx-rtl.dx-drawer-overlap.dx-drawer-slide
```

Reply:
Got it, thanks